### PR TITLE
delete missing data from volumes in one replica

### PR DESCRIPTION
**Pull request by kmlebedev**

### CODE REVIEW

File: weed/shell/command_volume_fsck.go [hunk 1]
```diff
@@ -33,7 +33,8 @@ func init() {
 }
 
 type commandVolumeFsck struct {
-	env *CommandEnv
+	env          *CommandEnv
+	forcePurging bool
```
Comment (chrislusf) [inline weed/shell/command_volume_fsck.go]: change this to a pointer?

File: weed/shell/command_volume_fsck.go [hunk 2]
```diff
@@ -68,6 +69,7 @@ func (c *commandVolumeFsck) Do(args []string, commandEnv *CommandEnv, writer io.
 	findMissingChunksInFilerPath := fsckCommand.String("findMissingChunksInFilerPath", "/", "used together with findMissingChunksInFiler")
 	findMissingChunksInVolumeId := fsckCommand.Int("findMissingChunksInVolumeId", 0, "used together with findMissingChunksInFiler")
 	applyPurging := fsckCommand.Bool("reallyDeleteFromVolume", false, "<expert only!> after detection, delete missing data from volumes / delete missing file entries from filer")
+	c.forcePurging = *fsckCommand.Bool("forcePurging", false, "delete missing data from volumes in one replica")
```
Comment (chrislusf) [inline weed/shell/command_volume_fsck.go]: need a better description here

Comment (kmlebedev) [reply weed/shell/command_volume_fsck.go]: > need a better description here

This is the best I've come up with :)

File: weed/shell/command_volume_fsck.go (hunk 1 above)
Comment (kmlebedev) [reply weed/shell/command_volume_fsck.go]: done

**Pull request merged**